web-next/src/routes/(root)/[handle]/(profile).tsx

```
@@ -0,0 +1,82 @@
+import type { ProfileLayoutQuery } from "./__generated__/ProfileLayoutQuery.graphql.ts";
+
+export const route = {
+  matchFilters: {
+    handle: /^@/,
+  },
+  preload(args) {
+    void loadProfileLayoutQuery(args.params.handle);
+  },
+} satisfies RouteDefinition;
+
+const ProfileLayoutQuery = graphql`
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ProfileLayoutQuery'. Consider to delete it or rename it.

'ProfileLayoutQuery' is defined here:


(lint/suspicious/noRedeclare)